src/markdown-pages/docs/ecosystem/ecosystem.md

```diff
@@ -0,0 +1,85 @@
+| **Credentials Store**                   | tbc |
+| **Certificate Store** | tbc                                                                                  |
+
+</details>
+
+
+<details>
+<summary>Galasa Servers</summary>
+
+|                                   |                                                                                                                          |
+| ------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| **Engine controller**                       | The Engine Controller is responsible for spinning up docker containers to execute individual Galasa automation runs.        |
+| **etcd**                       | Contains the Configuration Property Store (CPS), the Dynamic Status Store (DSS) and the Credentials Store (CREDs). The CPS, DSS are for use with by all users, the CREDs are for automation runs only.        |
+| **couchdb**                       | Contains the Result Archive Store (RAS) which contains the full record of an automated run.        |
+| **Resource Monitor**                       | The Resource Monitor service handles the cleaning up and management of resources in Galasa.        |
+| **Metrics Server**                       | A metrics server to indicate the health of the ecosystem to a Prometheus server. A Prometheus server scrapes and stores metrics from defined endpoints.       |
+| **Bootstrap Server**                       | The Galasa API server which includes the bootstrap       |
+| **Framework API Server**                       | The API server is an API mediation layer which sits between the Galasa framework and Galasa services, such as the properties services in the CPS. The API server acts as a central point from which to control the Galasa ecosystem.        |
+| **Manager API Server**                       | A Manger to Manger type API. The API is used for web services that can be called from the Galasa web UI or from external services like Jenkins. Visual Studio Code will use a lot of the Core APIs to communicate with the Galasa ecosystem, whilst Eclipse will use the SPI as we can load the framework into Eclipse.        |
+| **Web UI**                       | tbc        |
+| **LDAP authentication server (Optional)**                       | tbc        |
+
+</details>
+
+<details>
+<summary>Code Deployment</summary>
+
+|                                   |                                                                                                                          |
+| ------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| **Maven Repositories and OBRs**                       | tbc        |
+| **Nexus**                       | A Nexus server to provide an easy entry for deploying Maven artifacts to the ecosystem.        |
```

Comment:
A maven repo is required for an ecosystem to pull test material. We do bring up a nexus with the docker operator, but its not required if you have your own. Nexus is also nice as it can also host docker images, which if people ever build custom test engines, they can host them there